src/core/Requests/HttpRequest.js

```diff
@@ -0,0 +1,71 @@
+const AuthorizationStorage = require('./AuthorizationStorage');
+
+const baseUrl = 'localhost:8080'; // Zmienić podstawe baseUrl
+const authorization = AuthorizationStorage.getAuthorization();
+
+
+exports.GetRequest = (url) => {
+    fetch(`${baseUrl}/${url}`, {
+        method: 'GET',
+        authorization: authorization,
+        credentials: 'same-origin',
+        cache: 'no-cache',
+        contentType: 'application/json',
+    })
+        .then(resp => resolve(resp.body)
+    ).catch((error) => {
```

Comment:
Zwracajmy jsona, bądź obiekt z errorem. Usunąć więc emitowanie tutaj wydarzeń z Vue - to sprawa klienta, co  z tym zrobi. Podczas łapania wyjątku róbmy:
```
{errors: [e]}
```
po stronie klienty będziemy to rozpoznawać i pokazywać dialog.